slides/lec-2.qmd

```diff
@@ -20,12 +20,196 @@ format:
+
+## Steps
+
+- Think about a good folder structure 
+  - Split up 'read-only', 'human-generated', and 'project-generated' files
+- Create folder structure (main directory and sub directories)
+  - Add a landing page in the form of a README document
+  - Make the compendium executable (to automatically generate the results; optional)
+  - Make the compendium into a git repository (optional)
+- Add all files needed for reproducing the results of the project
+  - Avoid 'hard coded' parameters or human intervention in the execution
+- Make the compendium as clean and easy to use as possible
+  - Include a citation file and a LICENSE file with info on how it can be used
+- Publish your compendium
+  - E.g. on Zenodo (optional, more on this in the last course week)
+
+::: footer
+Adapted from [The Turing Way](https://the-turing-way.netlify.app/reproducible-research/compendia#checklist)
+::: 
+
+## Bonus: Preregister your simulation study
+
+![](images/lec-2/replisims.png)
+
+<!-- Siepe, B. S., Bartoš, F., Morris, T. P., Boulesteix, A.-L., Heck, D. W., & Pawel, S. (2024). Simulation studies for methodological research in psychology: A standardized template for planning, preregistration, and reporting. Psychological Methods. https://doi.org/10.1037/met0000695 -->
+
+::: footer
+[Siepe et al. (2024)](https://doi.org/10.1037/met0000695)
+
+[RepliSims](https://replisims.org)
+::: 
+
+
+## Next meeting
+
+Git en GitHub tutorial
+
+- Deelnemers leren samenwerken aan code middels git en GitHub. De basis voor deze tutorial is de groepsopdracht die in de ochtend is gemaakt.
+
+## References and further reading{.smaller}
+
+Baker, M. (2016). 1,500 scientists lift the lid on reproducibility. [https://www.nature.com/articles/533452a](https://www.nature.com/articles/533452a)
+
+Boulesteix, A.-L., Groenwold, R. H., Abrahamowicz, M., Binder, H., Briel, M., Hornung, R., Morris, T. P., Rahnenführer, J., & Sauerbrei, W. (2020). Introduction to statistical simulations in health research. BMJ Open, 10(12), e039921. [https://doi.org/10.1136/bmjopen-2020-039921](https://doi.org/10.1136/bmjopen-2020-039921)
+
+Bryan, J., & TAs, T. S. 545. (n.d.-a). Chapter 33 Why and how we automate data analyses + examples | STAT 545. Retrieved October 30, 2023, from [https://stat545.com/](https://stat545.com/)
+
+Checklist. (n.d.). Retrieved October 31, 2023, from [https://guide.esciencecenter.nl/#/best_practices/checklist](https://guide.esciencecenter.nl/#/best_practices/checklist)
+
+Checklist for a Software Management Plan. (n.d.). [https://doi.org/10.5281/zenodo.2159713](https://doi.org/10.5281/zenodo.2159713)
+
+Comment on Oberman & Vink: Should we fix or simulate the complete data in simulation studies evaluating missing data methods? - Morris—Biometrical Journal—Wiley Online Library. (n.d.). Retrieved October 30, 2023, from [https://onlinelibrary.wiley.com/doi/full/10.1002/bimj.202300085](https://onlinelibrary.wiley.com/doi/full/10.1002/bimj.202300085)
```

Comment:
This might be the wrong ref: the comment instead of the paper itself.

Oberman, H. I., & Vink, G. (2024). Toward a standardized evaluation of imputation methodology. Biometrical Journal, 66(1), 2200107. https://doi.org/10.1002/bimj.202200107

Reply:
I've removed the citation of the comment in any case. And your paper is still cited in lecture 1.